docs/architecture/adr-002-docs-structure.md

```diff
@@ -0,0 +1,270 @@
+
+#### docs/Explanation
+
+This folder will provide context for readers and are discursive in nature. Please see the [Divio Explanation page](https://documentation.divio.com/explanation/#) for more detail.
+
+- **docs/explanation/README.md** - This will orient the reader and explain the content. 
+
+##### docs/Explanation/ADR
+
+This is a folder that will track decisions regarding design and architecture (such as this). It will contain the following:
+
+- **docs/explanation/adr/README** - introduction to ADR
+- **docs/explanation/adr/PROCESS.md** which details how to raise ADRs
+- **docs/explanation/adr/adr-template.md** - template for raising ADR
+- **docs/explanation/adr/adr-{number}-{desc}.md** - an ADR document
+
+##### docs/Explanation/articles
+
+This SHALL BE a folder that contains a sub-folder for each published article. By published this COULD REFER to blog posts. The folder should be name such that it describes the article purpose. Each sub-folder SHALL CONTAIN all the content relevant to the article (e.g. images, bibliographies etc). These articles can be converted into pdf using pandoc. 
+
+In order to do this
+
+- There SHOULD BE a makefile with targets for calling pandoc - Note: the process for building pdf are not part of the commit or release processes, but ad-hoc
+- There SHOULD BE a latex template file that can create pdf files with consistent look and feel. This COULD BE the [Eisvogel template](https://github.com/Wandmalfarbe/pandoc-latex-template) with suitable modifications
+- The makefile and template should be independent of the article
+- THere SHOULD BE a README.md that describes how to use and build articles
+
+> **Note:** Explanations can come in other forms, particularly issue discussion and Pull Requests.
+
+#### docs/Tutorials
+
+As indicated in the overview, these SHALL BE documents that target beginners and will take a user step-by-step through a process with the aim of achieving some goal. Please see the [Divio tutorial page](https://documentation.divio.com/tutorials/) for detail.
+
+- There SHALL BE a folder for each tutorial - see the [Cosmos SDK tutorials](https://github.com/cosmos/sdk-tutorials) as an example
+- The folder SHALL CONTAIN all the content relevant for that tutorial. 
+- The content SHOULD BE consistent in format with [SDK tutorials](https://tutorials.cosmos.network/). 
+
+#### docs/How-To
+
+Compared to tutorials, [How-Tos](https://documentation.divio.com/how-to-guides/) are help an experienced reader solve a specific problem. These SHALL USE templates similar to the tutorials - see above.
+
+### Templates
+
+Documentation SHOULD USE templates for documentation such as module messages etc. 
+
+- [The good docs project](https://github.com/thegooddocsproject)
+- [Readme editor](https://readme.so/editor)
+
+
+#### Code Comments
+
+These are included as comments also form part of documentation. Comments SHALL FOLLOW recommendation as per [Conventional Comments](https://conventionalcomments.org/)
```

Comment:
```suggestion
PR review comments also form part of the documentation. Comments SHALL FOLLOW recommendation as per [Conventional Comments](https://conventionalcomments.org/)
```

Reply:
do we mean comments in the PR review process? or what are we talking about here?